src/AdvApi32.Desktop/AdvApi32.Helpers.cs

```diff
@@ -47,7 +160,7 @@ public static void InstallService(string servicePath, string serviceName, string
 
                     ServiceDescription descriptionStruct = new ServiceDescription
                     {
-                        lpDescription = serviceDescription
+                        lpDescription = lpDescription
                     };
 
                     IntPtr lpInfo = Marshal.AllocHGlobal(Marshal.SizeOf(descriptionStruct));
```

Comment:
(I know this code wasn't changed in this review but it might be interesting to fix this while you are in the file, otherwise no problem we'll create an issue and fix it later)

The code under that line is actually not really good :
* `AllocHGlobal` result is tested for null but there is no indication that it could ever happen
* Memory isn't always freed In case of exception raised by one of the P/Invoke, the `FreeHGlobal` should be in a `finally` block